kepub/converter.go

```diff
@@ -83,6 +88,43 @@ func (c *Converter) Convert(dir string) error {
 	return nil
 }
 
+func (c *Converter) GetHtmlFiles(dir string) ([]string, error) {
+	opfPath, err := FindOPF(dir)
+	if err != nil {
+		return []string{}, fmt.Errorf("transform opf: find: %w", err)
```

Comment:
You can return `nil` with the errors since a nil slice is treated as empty for pretty much all uses.